wordguess.rb

```diff
@@ -0,0 +1,109 @@
+# Emilce C  & Maria Velez
+# Ada C9 - Octos - Word-Guess
+# 12th - 14th Feb 2018
+# https://github.com/Ada-C9/Word-Guess
+
+class Art
+  attr_accessor :art
+
+  def initialize
+    @art = art
+  end
+
+  def show_art
+     show_art = "(=^ ◡ ^=)"
+     return show_art
+  end
+
+  def art_array
+    art_array = show_art.split("")
+    puts "#{art_array}"
+    return art_array
+  end
+
+  def eliminate_art
+    show_art.slice(0)
+  end
+
+
+end
+
+class Word
+  attr_accessor :guess_letter
+  attr_reader :selection_of_word, :create_underscore_array
+
+  def initialize
+    @word = selection_of_word
+    @replaced_underscore = create_underscore_array
+    @guesses_available = 9
+
+  end
+
+  def selection_of_word()
+    # body parts variable is set to an array of strings representing body parts
+    words = ["kitty", "paw", "litter", "leash", "tongue", "grumpy", "fierce", "tail", "whiskers", "claws"]
+    # randomly grabbing a word from body_parts array
+    random_word = words.sample
+
+  end
+
+  def create_underscore_array
+    random_word_length = @word.length
+    puts @word
+
+    underscore_array = Array.new(random_word_length, "-")
+
+    # putting contents of 'guess' using the join method (makes underscores print horizontally)
+    # user sees this
+    puts underscore_array.join("")
+    return underscore_array.join("")
+  end
+
+  def guess_letter
+    art = Art.new
+    # breaking up the word into separate letters so we can loop thru them
```

Comment:
Calling this method `guess_letter` is a little misleading - this method plays the entire game, top to bottom. Realizing when your method is getting away from you and becoming bigger than you intended is an important skill to practice.

One way to combat this tendency is to plan out how your code will be called before you start writing the class. You might start by building a driver loop down below:

```ruby
game = Word.new
while !game.is_over?
  game.print_state
  puts "Letter please"
  letter = gets.chomp
  game.guess_letter(letter)
end
```

This can serve as a sort of road map, limiting how big your methods need to be and keeping you on the right track.